Mathlib/Algebra/Group/Defs.lean

```diff
@@ -969,6 +969,7 @@ theorem zpow_natCast (a : G) : ∀ n : ℕ, a ^ (n : ℤ) = a ^ n
 #align zpow_of_nat zpow_natCast
 #align coe_nat_zsmul natCast_zsmul
 #align of_nat_zsmul natCast_zsmul
+#align matrix.zpow_nat_cast zpow_natCast
```

Comment:
This is surely nonsense?

Reply:
`Matrix.zpow_nat_cast` was just a special case of `zpow_natCast`, so I removed it. Somehow renaming it made the lemma be ambiguous in a bunch of places.

Reply:
Oh, then my objection is that this align should be in the file that it was deleted from